classes/class-mondu-gateway.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_Payment_Gateway' ) ) {
```

Comment:
Question: Why are we checking for WC_Payment_Gateway here, instead of MonduGateway?